packages/react-ui-core/src/Map/index.js

```diff
@@ -0,0 +1,2 @@
+export { default } from './Map'
+export { default as Mapbox } from './Map'
```

Comment:
we could not use Map because it exists on window and will not render via storybook. lets use 

`export { default as Mapbox } from './Mapbox'` and rename the file